src/scss/components/_serviceRichText.scss

```diff
@@ -0,0 +1,38 @@
+.richtext_title {
+  @include heading-4;
+
+  color: var(--brand-navy);
+  margin: 0;
+  padding-top: 6.4rem;
```

Comment:
Add this selector to apply the padding to all the `titles` but the first one
```
&:not(:first-of-type) {
    padding-top: 6.4rem;
 }
```

Reply:
ty!. done.